trackers/1major-with-surrrogate.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>1 major tracker loaded via script</title>
+
+  <script src="//static.doubleclick.net/tracker.js"></script>
```

Comment:
I don't think we surrogate this url? You should do `doubleclick.net/tag/js/gpt.js` or `doubleclick.net/instream/ad_status.js` IMO.

Also, `<title>` needs an update.